src/wallet/load.cpp

```diff
@@ -71,11 +71,16 @@ bool VerifyWallets(interfaces::Chain& chain)
 
         DatabaseOptions options;
         DatabaseStatus status;
+        options.require_existing = true;
         options.verify = true;
         bilingual_str error_string;
         if (!MakeWalletDatabase(wallet_file, options, status, error_string)) {
-            chain.initError(error_string);
-            return false;
+            if (status == DatabaseStatus::FAILED_NOT_FOUND) {
+                chain.initWarning(Untranslated(strprintf("Skipping -wallet path that doesn't exist. %s\n", error_string.original)));
```

Comment:
```suggestion
                chain.initWarning(strprintf(Untranslated("Skipping -wallet path that doesn't exist. %s\n"), error_string)));
```

Reply:
re: https://github.com/bitcoin/bitcoin/pull/20186#discussion_r507720604

> 

It would be nice if this worked, but it doesn't currently. We only overloaded tfm::format inutil/translation.h to substitute normal arguments into bilingual format strings, not bilingual arguments into bilingual format strings